development/compiling/compiling_for_android.rst

```diff
@@ -8,147 +8,154 @@ Compiling for Android
+To set those environment variables on Linux or macOS, use
 ``export ANDROID_HOME=/path/to/android-sdk`` and
-``export ANDROID_NDK_ROOT=/path/to/android-ndk``.
-Where /path/to/android-sdk and /path/to/android-ndk is the path where Android SDK
-and Android NDK are placed on your PC.
+``export ANDROID_NDK_ROOT=/path/to/android-ndk``
+where ``/path/to/android-sdk`` and ``/path/to/android-ndk`` point to
+the root of the SDK and NDK directories.
 
 Building the export templates
 -----------------------------
 
 Godot needs two export templates for Android: the optimized "release"
-template (`android_release.apk`) and the debug version (`android_debug.apk`).
-Compiling the standard export templates is done by calling scons with
+template (``android_release.apk``) and the debug template (``android_debug.apk``).
+Compiling the standard export templates is done by calling SCons with
 the following arguments:
 
--  Release template (used when exporting with "Debugging Enabled" OFF)
+-  Release template (used when exporting with **Debugging Enabled** unchecked)
 
 ::
 
-    C:\godot> scons platform=android target=release
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
+    scons platform=android target=release
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux
+    ./gradlew build
 
-(on Linux or macOS, execute the `gradlew` script with `./gradlew build`)
 
-The resulting APK is in:
-
-::
+The resulting APK will be located at ``bin/android_release.apk``.
 
-    bin\android_release.apk
-
--  Debug template (used when exporting with "Debugging Enabled" ON)
+-  Debug template (used when exporting with **Debugging Enabled** checked)
 
 ::
 
-    C:\godot> scons platform=android target=release_debug
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
-
-The resulting APK is in:
+    scons platform=android target=release_debug
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux/macOS
+    ./gradlew build
 
-::
 
-    bin\android_debug.apk
+The resulting APK will be located at ``bin/android_debug.apk``.
 
 Faster compilation
 ~~~~~~~~~~~~~~~~~~
 
-If you are on Unix or installed PyWin32 on Windows and have multiple CPU
-cores available, you can speed up the compilation by adding the ``-jX``
-argument to the SCons command, where ``X`` is the number of cores that you
-want to allocate to the compilation, e.g. ``scons -j4``.
-
+If you have multiple CPU cores available, you can speed up compilation
+by adding the ``-jX`` argument to the SCons command, where ``X`` is the
+number of cores you want to allocate to building, e.g. ``scons -j4``.
 
 Adding support for x86 devices
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you also want to include support for x86 devices, run the scons command
+If you also want to include support for x86 devices, run the SCons command
```

Comment:
This section (and the first one about the "default" build) should be rewritten as there are now three targets to take into account, and people should build at least `armv7` and `arm64-v8` (and optionally `x86` if they want).

So it should be made clearer from the start that you build Godot for all your target platforms first, and *then* you run `gradlew build` to generate the APK.